src/index.njk

```diff
@@ -93,6 +93,12 @@ templateEngineOverride: njk
             <footer class="mt-3">House of Commons, Science, Innovation and Technology Committee</footer>
           </blockquote>
         </div>
+        <div class="carousel-item">
+          <blockquote class="iai-homepage-quote">
+            <p class="text-xl">“Artificial intelligence technologies offer the potential to transform the UK’s economic landscape and improve people’s lives across the country”</p>
+            <footer class="mt-3">David Knott <span aria-hidden="true">|</span> Chief Technology Officer <span aria-hidden="true">|</span> UK Government</footer>
```

Comment:
It's a relatively minor thing (especially if the content has been signed-off), but the sources seem to be cited in a different format for each quote. If nothing else, I wonder if changing the second to `House of Commons | Science, Innovation and Technology Committee` would seem more consistent

Reply:
I _think_ it's one committee - i.e. "The House of Commons Science, Innovation and Technology Comittee". If so, it doesn't make sense to split it up. But if I'm wrong on that, then it does make sense to change!
Whereas the others have clear separations of person, role, organisation, date.

Reply:
That makes sense. I think we might need to get rid of the comma after 'Commons', for readability